--- Makefile.orig	2025-05-13 23:00:53
+++ Makefile	2025-05-13 23:16:47
@@ -1,8 +1,8 @@
 all: build
 
-GIT_COMMIT:=$(shell git rev-list -1 HEAD)
-GIT_LAST_TAG:=$(shell git describe --abbrev=0 --tags 2>/dev/null || true)
-GIT_EXACT_TAG:=$(shell git name-rev --name-only --tags HEAD)
+GIT_COMMIT:=@@GIT_COMMIT@@
+GIT_LAST_TAG:=@@GIT_COMMIT@@
+GIT_EXACT_TAG:=@@GIT_COMMIT@@
 UNAME_S := $(shell uname -s)
 XARGS:=xargs -r
 ifeq ($(UNAME_S),Darwin)
@@ -25,7 +25,7 @@
 		xargs -I NAME printf '\t%-20s %s\n' "NAME" "nix build .#checks.linux.NAME"
 
 .PHONY: build
-build:
+build: pack-webui
 	go generate
 	go build -ldflags "$(LDFLAGS)" .
 
@@ -59,8 +59,12 @@
 test:
 	go test -v -bench=. ./...
 
+.PHONY: npm-install
+npm-install:
+	npm --prefix webui install
+
 .PHONY: pack-webui
-pack-webui:
+pack-webui: npm-install
 	npm run --prefix webui build
 	go run webui/pack_webui.go
 
